package.json

```diff
@@ -15,8 +15,8 @@
     "@babel/parser": "^7.0.0",
     "@babel/traverse": "^7.0.0",
     "@babel/types": "^7.0.0",
-    "eslint-scope": "3.7.1",
-    "eslint-visitor-keys": "^1.0.0"
+    "eslint-visitor-keys": "^1.0.0",
```

Comment:
Ideally we should not pin `eslint-visitor-keys` either. But `eslint` does not import `eslint-visitory-keys` until 4.14.0. As we have specified eslint peerDependencies to `>=4.12.1`, I would rather leave it here and then we can remove it on v11.